bd_spot_wrapper/spot_wrapper/spot_qr_detector.py

```diff
@@ -156,7 +158,8 @@ def get_avg_spotWorld_T_marker_HAND(
             print(f"Iterating - {len(marker_position_from_dock_list)}")
             is_marker_detected_from_hand_cam = False
             img_response_hand = self.spot.get_hand_image()
```

Comment:
Just curious: What does the return type for this look like?

Reply:
`get_hand_image()` returns a list of 2 elements.
Both elements are BD :: `ImageResponse` objects. First object is for rgb, second object is for depth in rgb frame.

Updated.

Reply:
More info about ImageResponse object can be found here - https://dev.bostondynamics.com/protos/bosdyn/api/proto_reference#bosdyn-api-ImageResponse

Reply:
could you add this link as a comment for n00bs like me :)

Reply:
Will do, @rutadesai

Reply:
Added the changes @Ruta Desai